kf_lib_data_ingest/target_apis/kids_first.py

```diff
@@ -418,3 +418,100 @@ def validate():
                 f'Links schema for "{target_concept}" has an '
                 f"incorrect or null mapping, {link_key}: {tgt_id_mapping}"
             )
+
+
+# TODO: This should be merged into the target API config!
```

Comment:
What do you mean by that? This method should go in target api config?

Reply:
refactoring the target api config should make this additional section unnecessary